CoronaCare/src/screens/AfterLogin.js

```diff
@@ -27,7 +27,7 @@ export default class Home extends Component {
 
     render() {
         return (
-            <SafeAreaView style ={styles.bkgrdContainer}>
+            <View style ={styles.bkgrdContainer}>
```

Comment:
Whats the difference between View and SafeAreaView?

Reply:
Safe area view is meant for devices with a top notch for instance iphone x onwards, they have the top notch for face id, safe area view ensures that the component avoids displaying anything in the top notch automatically.